nbformat/validator.py

```diff
@@ -230,7 +233,7 @@ def better_validation_error(error, version, version_minor):
 
 
 def validate(nbdict=None, ref=None, version=None, version_minor=None,
-             relax_add_props=False, nbjson=None):
+             relax_add_props=False, nbjson=None, repair_invalid_cell_ids=True):
```

Comment:
Let's use `repair_duplicate_cell_ids` as we're not replacing ids that don't conform to the spec's regex. After that change LGTM